server/utils/nunjucksSetup.js

```diff
@@ -4,8 +4,8 @@ module.exports = (app, path) => {
   const njkEnv = nunjucks.configure(
     [
       path.join(__dirname, '../../server/views'),
-      'node_modules/govuk-frontend/',
-      'node_modules/govuk-frontend/components/',
+      'node_modules/govuk-frontend/govuk/',
```

Comment:
is this correct?
https://github.com/alphagov/govuk-frontend/releases states
>Change the list of paths in nunjucks.configure so that the only GOV.UK Frontend path is node_modules/govuk-frontend/:

Reply:
As discussed, have changed to node_modules/govuk-frontend/